src/Umbraco.ModelsBuilder/Umbraco/LiveModelsProvider.cs

```diff
@@ -133,9 +127,7 @@ public void Dispose()
 
         public static void Install()
         {
-            if (!LiveModelsProvider.IsEnabled)
```

Comment:
Seems odd this was removed, i'll update with recent changes. The module should not be enabled if pure live is not enabled.